plugin.video.twitch/changelog.txt

```diff
@@ -1,5 +1,7 @@
+1.4.5
+- Added compatibility for new Twitch encoder names.
 1.4.4
-- HTTPS fix for Linux users + Kodi Official repository compatibility fixes.
+- HTTPS fix for Linux users + Kodi Offical repository compatibility fixes.
```

Comment:
Heh, that was an unintentional change. Thanks for the heads up.

Ill take a look at the thread.

Thanks for merging.